clang/unittests/Format/FormatTest.cpp

```diff
@@ -7951,6 +7951,24 @@ TEST_F(FormatTest, AllowAllArgumentsOnNextLineDontAlign) {
                Input, Style);
 }
 
+TEST_F(FormatTest, BreakFunctionDefinitionParameters) {
```

Comment:
As I understand you want this, so that you can put all arguments on a single line. My advice is put that in the tests also, so no one could break it by accident.